src/frontend_workspaces/frontend/src/carbon-chat/CarbonChat.css

```
@@ -434,3 +434,250 @@
+  color: #57606a;
+  font-size: 12px;
+  line-height: 1.4;
+}
+
+/* CUGA does not currently have an app-wide dark mode. The slash-dropdown
+   dark-mode rules previously living here were no-op'd to avoid the visual
+   inconsistency of dark chips inside an otherwise light UI. Restore them
+   (selectors: .cuga-slash-dropdown, .cuga-slash-dropdown__status,
+   .cuga-slash-option*) if/when a dark mode is introduced. */
+@media (prefers-color-scheme: dark) {
+}
```

Stylelint (17.11.0)

[error] Empty block (block-no-empty)

(block-no-empty)